.github/workflows/cache.yml

```
@@ -0,0 +1,36 @@
+name: Cache Keyman API
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Fetch from Keyman API
+        run: |
+          curl https://api.keyman.com/search?q= > src/app/output.json
+
+      - name: Commit JSON output
+        id: commit
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "github-actions"
+          git add --all
+          if [-z "$(git status --porcelain)"]; then
```

`[-z` is invalid syntax because bash is a pile of hacks and `[` is the name of a PROGRAM (which is located at `/usr/bin/[`).  You need a space between the `[` (the program name and `-z`, its first argument.

```suggestion
          if [ -z "$(git status --porcelain)"]; then
```

Note that `[[` is bash-only syntax that makes some `if` statements in bash slightly easier. See: https://jvns.ca/blog/2017/03/26/bash-quirks/#if-statements:~:text=if%20statements,-If

See: https://superuser.com/a/334576/711047